shared/src/test/scala/crystal/PotSpec.scala

```diff
@@ -40,20 +40,29 @@ class PotSpec extends DisciplineSuite {
 
   property("Pot[Int].toOption: Pending(_) is None") {
     forAll((l: Long) => Pending(l).toOption.isEmpty)
+  }
 
+  property("Pot[Int].toOption: Pending(_) isPending") {
+    forAll((l: Long) => Pending(l).isPending)
```

Comment:
Maybe we can also test for `&& !isReady && !isError`, and similar in the other cases below.